lint/witnesslint/__init__.py

```diff
@@ -0,0 +1,7 @@
+# This file is part of sv-witnesses repository: https://github.com/sosy-lab/sv-witnesses
+#
+# SPDX-FileCopyrightText: 2021 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+__version__ = "1.4"
```

Comment:
Why does this PR still change the version number instead of simply keeping it at `1.3-dev`?

And now, if we do not rewrite the history of this PR, we will have several different commits in the project history where the linter will claim to be version 1.4, with potentially different behavior (if version 1.4 is actually released at some point in the future, because the next version should be 1.3?). This is exactly what should be avoided by bumping the version only directly before and after a release, as I suggested before.

Reply:
I was under the impression that we wanted to release the package immediately once this PR is merged. But I did forget that one could now find this version number on multiple commits with different behaviors. Then again, the same is true for the version numbers we used so far, so this does not worsen the situation much. We can publish the first release as version 1.5, and from then on only bump the version directly before/after new releases to avoid this problem in the future.

Reply:
I would leave out versions 1.3 and 1.4 then and change this PR to set the version to 1.5-dev, then once the PR is merged we can do the release by removing the -dev, publishing, then bumping to 1.6-dev

Rationale: going back to 1.3 will be confusing with the commits here containing 1.4; If we leave out 1.3, we might as well leave out 1.4 as well, there is no reason not to jump to 1.5-dev here.